components/gateway/internal/auditlog/factory.go

```diff
@@ -0,0 +1,70 @@
+package auditlog
+
+import (
+	"time"
+
+	"github.com/kyma-incubator/compass/components/gateway/internal/auditlog/model"
+)
+
+const (
+	LogFormatDate  = "2006-01-02T15:04:05.999Z"
+	UserVariable   = "$USER"
```

Comment:
please add somewhere the purpose of this consts: that they are automatically filled by external service, they are used probably only in OAuth flow. I am also wondering, if these variables should not be configurable (with current values as a default), I can imagine situation that External Audit Service is changing the name of those variables, and then we have to build a new image of our component, instead of just changing env variables.